### PR TITLE
Cypress: Add VM test case

### PR DESCRIPTION
**Pull request by n313893254**
Manual test case: 
- https://harvester.github.io/tests/manual/virtual-machines/cpu_overcommit/
- https://harvester.github.io/tests/manual/virtual-machines/create-a-new-vm-and-add-enable-usb-tablet-option/

### CODE REVIEW

File: cypress/testcases/virtualmachines/advanced.spec.ts [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      memory: '4',
+      image: 'ubuntu-18.04-server-cloudimg-amd64.img',
+      usbTablet: true,
+    }
+    vms.setValue(value);
+
+    vms.save();
+  }) 
+
+  it('Step 2: Check config', () => {
```
Comment (lanfon72) [inline cypress/testcases/virtualmachines/advanced.spec.ts:32]: based on [the practice](https://docs.cypress.io/guides/references/best-practices#Having-tests-rely-on-the-state-of-previous-tests), _steps_ should be put in the same `specify / it` testing.

File: cypress/testcases/virtualmachines/cpu-overcommit.spec.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    settings.goToEditSetting('overcommit-config')
+
+    settings.setOvercommit({
+      cpu: '2000',
+    })
+
+    settings.save()
+  })
+
+  it('Step2: Create VM && Check VM request', () => {
```
Comment (lanfon72) [inline cypress/testcases/virtualmachines/cpu-overcommit.spec.ts:35]: same problem as previous review

**Review by lanfon72 — CHANGES_REQUESTED** [review]

**Comment by n313893254** [comment]
@lanfon72 Done, thanks for your review. PTAL

**Review by lanfon72 — APPROVED** [review]
LGTM

**Pull request merged**